src/pages/Contents/Contents.js

```diff
@@ -1,11 +1,128 @@
 import React, { Component } from "react";
 import "./Contents.scss";
+import { CaretDownOutlined, PlusOutlined } from "@ant-design/icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { pinkFlag } from "./ActorProfile/data";
 
 class Contents extends Component {
+  constructor() {
+    super();
+    this.state = {
+      addBtnClicked: false,
+      dropdownBtnClicked: false,
+      dropdownBtnColorChanged: false,
+      commentBtnClicked: false
+    };
+  }
   render() {
     return (
       <>
-        <div>sdf</div>
+        <section>
+          <div className="header_box">
+            <div className="dark_box"></div>
```

Comment:
content가 없는 태그는 아래와 같이 selfclosing태그로 작성해주세요! 

```suggestion
            <div className="dark_box" />
```

Reply:
넵!